website/content/maintainers/community/contributor-growth-framework/open-source-roadmaps.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Roadmaps as a way to encourage contributions"
+date: 2023-06-28
+description: >
+  A guide to open-source project roadmaps.
+---
+
+# Importance of Roadmaps in Open Source Projects
+
+A roadmap is a strategic plan that outlines the goals and direction of an open source project. For open source projects, having a well-defined roadmap is crucial.
```

Comment:
Could we include characteristics of a roadmap or more specifically what the benefit of strategic planning is to projects?  Some projects may not use the traditional GitHub implementation of a "roadmap" so I want to ensure we are outcome oriented and not prescriptive of the tool/mechanism to achieve that outcome.

Reply:
Added a section on "Roadmaps bring a stratigic advantage"
Maybe I missed the point at 2am in the morning. :smile:

Reply:
I don't think I was clear in my comment, apologies. I meant that strategic planning is important for projects to perform and roadmaps are an essential tool to capture that planning so adopters, contributors, and others understand where a project is headed, when it plans to get there, and how it will do it.

Reply:
I added a bullet point for "strategic planning"